docs/conf.py

```diff
@@ -1,10 +1,11 @@
 import os
 import sys
 
+from praw import __version__
+
 sys.path.insert(0, ".")
 sys.path.insert(1, "..")
 
-from praw import __version__
```

Comment:
I thought the order here was important. If it's not, I don't there that there is any need for these inserts. Tests are passing, but I'm not sure why.

Reply:
This line exists because doing sphinx build in the `docs` directory would cause an error, when the global praw conflicted with the local praw. These lines can be removed entirely.

Reply:
Let's give it a try and see what happens.

Reply:
I was going to remove these lines, but it has already been committed. I'll start another PR for it.